### PR TITLE
Approve PBA submit functionality

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3680

### Change description ###
Approve PBA submit functionality - Happy Path

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<ng-container *ngIf="org; else noOrg">
+  <h1 class="govuk-heading-xl">Confirm your decision</h1>
+  <app-pba-account-decision *ngFor="let pba of org.pbaNumber; let i = index" [pbaNumber]="pba"
+    [accountName]="org.accountDetails[i]?.account_name" [organisationId]="org.organisationId"
+    [decision]="newPBAs.get(pba) === 'approve' ? 'Approve the account' : 'Reject the account'">
```
Comment (paddybasi) [inline src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.html:5]: maybe could move this logic into the component, makes it testable etc

**Review by paddybasi — APPROVED** [review]

File: src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.html (hunk 1 above)
Comment (paulhowes-HMCTS) [reply src/org-manager/components/new-pbas-confirm/new-pbas-confirm.component.html:5]: Fair comment. It's currently getting this out of the ngrx store and I didn't want to refactor it at this stage as moving forward (one of the next few tickets) we are going to be making calls to the node layer for this information, at which point the logic will be in the component and will be readily testable.

**Review by tarikdogan-hmcts — CHANGES_REQUESTED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

File: src/org-manager/containers/new-pbas/new-pbas.component.spec.ts [hunk 2]
```diff
@@ -18,23 +18,23 @@ describe('NewPBAsComponent', () => {
...
+      ],
+      declarations: [
+        NewPBAsComponent
+      ],
+      schemas: [
+        CUSTOM_ELEMENTS_SCHEMA
+      ],
+      providers: [
+      ]
     }).compileComponents();
```
[the remark this answers is not in the document]
Comment (paulhowes-HMCTS) [reply src/org-manager/containers/new-pbas/new-pbas.component.spec.ts:38]: Removed.

File: src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts [hunk 3]
```diff
@@ -13,7 +14,10 @@ describe('NewPBAsInfoComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [RouterModule, RouterTestingModule.withRoutes([])],
-      declarations: [NewPBAsInfoComponent, OrganisationAddressComponent, PBAAccountApprovalComponent]
+      declarations: [NewPBAsInfoComponent, OrganisationAddressComponent, PBAAccountApprovalComponent],
+      schemas: [
+        CUSTOM_ELEMENTS_SCHEMA
+      ],
```
[the remark this answers is not in the document]
Comment (paulhowes-HMCTS) [reply src/org-manager/components/new-pbas-info/new-pbas-info.component.spec.ts:20]: I've removed the comma.

**Review by tarikdogan-hmcts — APPROVED** [review]

**Pull request merged**